docs/user-guide/configure.md

```diff
@@ -74,9 +66,18 @@ You can add any number of keys in the object. For example, you can:
 }
 ```
 
+Some rules and options accept regex. You can enforce these common cases:
+
+- kebab-case: `^([a-z][a-z0-9]*)(-[a-z0-9]+)*$`
```

Comment:
[suggest] Because Prettier doesn't handle an underscore (_) character properly in some cases, `prettier --check` (`npm run lint:formatting`) fails. See https://github.com/prettier/prettier/issues/6112

Ignoring the list node should fix the problem as below:

```suggestion
<!-- prettier-ignore -->
- kebab-case: `^([a-z][a-z0-9]*)(-[a-z0-9]+)*$`
```